subsys/bluetooth/Kconfig

```diff
@@ -172,6 +172,13 @@ config BT_PER_ADV_SYNC_TRANSFER_RECEIVER
 config BT_PER_ADV_SYNC_TRANSFER_SENDER
 	bool "Periodic Advertising Sync Transfer sender"
 	depends on !BT_CTLR || BT_CTLR_SYNC_TRANSFER_SENDER_SUPPORT
+
+config BT_SCA_UPDATE
+	bool "Sleep Clock Accuracy Update"
+	default y
```

Comment:
Is it intentional that BT_SCA_UPDATE is default y?

Reply:
Good question;-)
I opted to keep default y, when I created the config, but I guess it should probably be tied to ISO configs in some way. Not sure how this is done best. 
Could be 'default y if BT_ISO_CENTRAL || BT_ISO_PERIPHERAL' similar to what I did for BT_CTLR_SCA_UPDATE

Reply:
Done